api/python/quilt3/packages.py

```diff
@@ -436,12 +438,26 @@ def install(cls, name, registry=None, top_hash=None, dest=None, dest_registry=No
                     f"'build' instead."
                 )
 
+        subpkg_key_index = name.find('/', name.find('/') + 1)
```

Comment:
Searching for '/' in the string is a bit confusing and seems a bit fragile. Can you use a regex here? Preferably update the `quilt3.util.PACKAGE_NAME_FORMAT` to allow an optional sub-package path. You should be able to use that regex to separate the package name and sub-package path.

Reply:
> Preferably update the `quilt3.util.PACKAGE_NAME_FORMAT` to allow an optional sub-package path. You should be able to use that regex to separate the package name and sub-package path.

I think we should add the separate regex as we don't allow sub-package in other places, makes sense?

Reply:
Yes, please proceed with the separate regex. You can create e.g. `quilt3.util.SUBPACKAGE_NAME_FORMAT`.